File: pgm/ABC_Dart/ABC_366_A.dart
// import 'dart:collection';
import 'dart:io';
import 'dart:math';
// import 'dart:typed_data';

void main() {
  final [int N, int T, int A] = stdin
      .readLineSync()!
      .split(' ')
      .map(int.parse)
      .toList();
  const String Yes = "Yes";
  const String No = "No";
  const int two = 2;
  if (max(T, A) > N ~/ two) {
    print(Yes);
  } else {
    print(No);
  }
  exit(0);
}

File: pgm/ABC_Kotlin/ABC_366_A.kt
// import java.time.*

import kotlin.math.*

const val Yes: String = "Yes"
const val No: String = "No"
const val space: String = " "
const val two: Int = 2

fun main() {
    val (N: Int, T: Int, A: Int) = readln().split(space).map { it.toInt() }
    if (max(T, A) > N / two) {
        println(Yes)
    } else {
        println(No)
    }
    kotlin.system.exitProcess(0)
}